tests/scripts/generate_tool_versions/conftest.py

```
@@ -0,0 +1,146 @@
+            {
+                "devDependencies": {
+                    "oxfmt": "^0.43.0",
+                    "@astrojs/check": "0.9.8",
+                },
+            },
+            indent=2,
+        ),
+    )
+
+    (tmp_path / "pyproject.toml").write_text(
+        "[project]\n" 'name = "fake"\n' 'dependencies = ["pytest>=9.0.3"]\n',
```

Ruff (0.15.12)

[warning] Implicitly concatenated string literals on one line

(ISC001)